api/geth_backend.go

```diff
@@ -628,6 +628,11 @@ func (b *GethStatusBackend) startNode(config *params.NodeConfig) (err error) {
 	}); err != nil {
 		return
 	}
+	wallet, err := b.statusNode.WalletService()
+	if err != nil {
+		return err
+	}
+	wallet.SetClient(b.statusNode.RPCClient().Ethclient())
```

Comment:
The var `wallet` collides with the package `github.com/status-im/status-go/services/wallet`, although this code is compliable it makes any reference to "wallet" in this function ambiguous. An easy fix is to rename the var to `walletService` or `w` or something cooler.

Reply:
fixed